### PR TITLE
Feature/syslumenn news

### PR DESCRIPTION
**Pull request by Ingimarsson**
# Syslumenn News

## What

This PR adds a news section to the Sýslumenn page by extending the existing `News` model. Generic tags are used to select where the news entry should be displayed. Note that frontpage news will need to have the "Stafrænt Ísland" tag from now on.

## Why

Organizations need to be able to post news, with the option to show important news on the front page as well (this is done by choosing the appropriate tags)

## Screenshots

image

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/components/NewsArticle/NewsArticle.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react'
```
Comment (barabrian) [inline apps/web/components/NewsArticle/NewsArticle.tsx:1]: Is this component specific for institutions? if so I'm just wondering if the name is too generic and could lead to some misunderstanding

**Review by barabrian — COMMENTED** [review]
This looks like a lot of visual additions and updates, it would help a lot to see some screen shots or gifs for visual references

File: apps/web/screens/Syslumenn/Home.tsx [hunk 2]
```diff
@@ -99,17 +112,47 @@ const Home: Screen<HomeProps> = ({ organizationPage, namespace }) => {
...
         </GridRow>
       </GridContainer>
+      <Section
+        paddingTop={[8, 8, 6]}
+        paddingBottom={[8, 8, 6]}
+        background="purple100"
+        aria-labelledby="latestNewsTitle"
+      >
+        <LatestNewsSection
+          label="Fréttir og tilkynningar"
```
[the remark this answers is not in the document]
Comment (Ingimarsson) [reply apps/web/screens/Syslumenn/Home.tsx]: Fixed.

File: apps/web/screens/Syslumenn/NewsList.tsx [hunk 3]
```diff
@@ -0,0 +1,425 @@
...
+    },
+    {
+      title: organizationPage.title,
+      href: linkResolver('organizationpage', [organizationPage.slug]).href,
+      typename: 'organizationpage',
+    },
+  ]
+
+  const sidebar = (
+    <Stack space={3}>
```
[the remark this answers is not in the document]
Comment (Ingimarsson) [reply apps/web/screens/Syslumenn/NewsList.tsx]: Fixed.

File: apps/web/screens/Syslumenn/NewsList.tsx [hunk 4]
```diff
@@ -0,0 +1,425 @@
...
+              color="purple400"
+            />
+            {selectedYear && (
+              <div>
+                <Link href={makeHref(selectedYear)}>
+                  <Text as="span">{allMonthsString}</Text>
+                </Link>
+                <Text as="span">
+                  {selectedMonth === undefined && <Bullet align="right" />}
+                </Text>
```
[the remark this answers is not in the document]
Comment (Ingimarsson) [reply apps/web/screens/Syslumenn/NewsList.tsx]: These Bullet elements didn't really make sense so I removed this block.

File: apps/web/components/NewsArticle/NewsArticle.tsx (hunk 1 above)
Comment (Ingimarsson) [reply apps/web/components/NewsArticle/NewsArticle.tsx]: I think the idea was to use the same wrapper for both frontpage news and organization news in the future. But for now I've moved it under `components/organization/NewsArticle`

**Comment by Ingimarsson** [comment]
I've added screenshots to the PR :)

**Review by barabrian — APPROVED** [review]
LGTM

**Pull request merged**